### ISSUE TITLE
Shorthand error when using variables and math

### ISSUE THREAD

**Issue by starryeyez024** [brigade/scss-lint#566]
An error is thrown when math is used within a property that allows for multiple values, such as padding. Example scss:

    .foo {
	padding: ($fixed-header-offset + 30) 0 0 0;
    }

Error:
Shorthand: Shorthand form for property `padding` should be written more concisely as `$fixed-header-offset + 30 0 0` instead of `$fixed-header-offset + 30 0 0 0`

**Comment by sds** [comment]
Hey @starryeyez024,

The warning in this case is legitimate. In CSS you can use what are called "shorthand" forms for certain properties like `margin` and `padding`.

For example, instead of specifying:

```scss
margin: X Y 0 Y;
```

...you can just write:

```scss
margin: X Y 0;
```

...since the second `Y` is redundant. This is the case in your example above. Hope that helps!

**Comment by starryeyez024** [comment]
Ah, I couldn't see the forest for the trees. Thanks!!
